examples/island/Island.cpp

```diff
@@ -182,17 +182,17 @@ int main()
         for (sf::Event event; window.pollEvent(event);)
         {
             // Window closed or escape key pressed: exit
-            if ((event.type == sf::Event::Closed) ||
-                ((event.type == sf::Event::KeyPressed) && (event.key.code == sf::Keyboard::Key::Escape)))
+            if (event.is<sf::Event::Closed>() || (event.is<sf::Event::KeyPressed>() &&
+                                                  event.getIf<sf::Event::KeyPressed>()->code == sf::Keyboard::Key::Escape))
```

Comment:
This is an interesting proposal of the formatter... 😬 

One way to keep it more explicit would be to add something like `//.` at the end of the first line, after the `||`. Not sure if it's worth it... Either way, definitely not a blocker, feel free to ignore 🙂